docs/reference/transform/apis/get-transform-stats.asciidoc

```diff
@@ -72,9 +72,81 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=size-transforms]
 [[get-transform-stats-response]]
 ==== {api-response-body-title}
 
-`transforms`::
-  (array) An array of statistics objects for {transforms}, which are
-  sorted by the `id` value in ascending order.
+The API returns an array of statistics objects for {transforms}, which are
+sorted by the `id` value in ascending order. All of these properties are
+informational; you cannot update their values.
+
+`checkpointing`::
+(object) Contains statistics about <<transform-checkpoints,checkpoints>>.
+`checkpointing`.`changes_last_detected_at`:::
+(date) The timestamp when changes were last detected in the source indices.
+`checkpointing`.`last`:::
+(object) Contains statistics about the last completed checkpoint. 
+`checkpointing`.`last`.`checkpoint`::::
+(TBD) A unique identifier for the checkpoint.
+`checkpointing`.`last`.`time_upper_bound_millis`::::
+(date) TBD
```

Comment:
> ... timestamp until data has been processed...

Thanks for the feedback @hendrikmuhs !  I'm not sure I understand this description yet, however.  Is it the duration of the checkpoint?

Reply:
Think of a continuous transform, where you source indexing gets new data in, so the destination/transformed index runs always behind the source. time_upper_bound marks the timestamp until all data from source has been processed into dest. So it's not a duration its an endmarker until data has been processed. 

There is also timestamp and it might seem like the same thing, but timestamp is the time the checkpoint has been created, time_upper_bound has to take the delay into account. So normally `time_upper_bound = timestamp - delay`. (However in future this might change, that's why `timestamp` and `time_upper_bound` are separate fields)

Reply:
Thanks, I've drafted changes to those two descriptions. If they still need tweaking, please just let me know.